### ISSUE TITLE
Small improvements

### ISSUE THREAD

**Issue by pixelastic** [urre/searchable-awesome-public-datasets#1]
Hello,

Great work on this awesome search. This is a very good idea and will come in handy in the future!

I have a few feedback to make it even better :)

## Increase the contrast
![image](https://user-images.githubusercontent.com/283419/43643749-a128a02e-972c-11e8-8723-4dcd164719c2.png)
The contrast on the grey seems to be too low and makes reading the content pretty hard to my eyes

## Cursor on the tags
I would suggest adding a `:hover` state to the tag selection. Maybe changing the color and/or just changing the cursor to a pointer. It will make it more obvious that you can click on the tags name (and not only the checkboxes)

## Highlighting
I would also suggest adding some highlighting to the results, to show more clearly why they match and making scanning them

## Pictures
You could also improve the rendering by adding a thumbnail of the website result. Most website have a `meta[property=og:image]` tag that holds a url to such an image. I find it much easier to browse a list of links when they have such images attached to them, it makes scanning them way easier IMO.

## Algolia link
![image](https://user-images.githubusercontent.com/283419/43643838-d8a9c820-972c-11e8-87be-ede6916b50d0.png)

Can I also ask you to add a link on the "Search by Algolia" image to point to the Algolia website. This will help more people know what we do :)

Keep up the good work :)

**Comment by urre** [comment]
Hey!

I've fixed most of the stuff now.

👁 Contrast has been increased on the urls
🖊 Highlighting is now added
🔗 Algolia link added
👀 Added hover effect to the category checkbox labels, also coloured when active now
👀 Added hover effect to the refinements category delete button


🖼 Pictures is a good idea. I'll look into that in the future when changing the scraper/indexer.

Thanks!

https://searchable-awesome-public-datasets.now.sh/
